mappings/net/minecraft/resource/NamespaceResourceManager.mapping

```diff
@@ -1,4 +1,10 @@
 CLASS net/minecraft/class_3294 net/minecraft/resource/NamespaceResourceManager
+	COMMENT A resource manager that only loads resources for a specific namespace.
+	COMMENT This is an implementation detail for {@link ResourceManagerImpl}, based
+	COMMENT on the heuristic that most namespaces are only defined in few resource
+	COMMENT packs, making loading in those namespaces faster.
+	COMMENT
+	COMMENT @see ResourceManagerImpl
```

Comment:
Did you mean `LifecycledResourceManagerImpl`? I can't find `ResourceManagerImpl`.

Reply:
```suggestion
	COMMENT @see LifecycledResourceManagerImpl
```